Feature/i229 landing page

martyn-w opened the pull request:
# Landing page with restricted visibility of metadata
Addresses issue 229
Merges into #158

nabeta commented:
Are the changes in this pull request included in ``feature/i230-authorisation`` branch? If so, is it going to be closed?
https://github.com/antleaf/nims-hyrax/pull/158

martyn-w commented:
Hi @nabeta, this branch was originally set to merge into `feature/i230-authorisation` as it required that code. Now that `feature/i230-authorisation` has been merged into `develop`, I have rebased this PR against develop, and also fixed a couple of merge conflicts. 

Please test and merge into develop.

nabeta commented:
I fixed the conflict with https://github.com/antleaf/nims-hyrax/pull/169 and this PR.
